### ISSUE TITLE
Make all the links opened in a new tab

### ISSUE THREAD

**Issue by devienyogiana** [kulkultech/open-source#94]
I think it would be more convenient for the user to open all the links in a new tab.

**Comment by empeje** [comment]
That's an excellent idea. Could you point out which links should be in the new tab which link should be in the same tab?

**Comment by devienyogiana** [comment]
I suppose all the links from step 0 to 9 should be in a new tab since the aspiring interns will navigate back and forth to those pages.
All the links in the header page such as FAQ, Member can be in the same tab

**Comment by empeje** [comment]
Sounds like a plan. You can go ahead implementing it in a pull request.

**Comment by devienyogiana** [comment]
After doing some research, it turns out that it's not possible to do this without adding some html code.

**Comment by empeje** [comment]
Yes, that's one of the challenges of using Markdown purely. Please explore other potential improvement and create a new issue.

**Comment by Divyansh1011** [comment]
we can try tackling this  problem by using 

``` [link](url){:target="_blank"}
```

**Comment by Divyansh1011** [comment]
we can try tackling this problem by using this 

```
[link](url){:target="_blank"}
```

**Comment by Divyansh1011** [comment]
For more reference https://stackoverflow.com/questions/4425198/can-i-create-links-with-target-blank-in-markdown

**Comment by empeje** [comment]
That's a good idea @Divyansh1011, could you please try to demonstrate in a pull request?
